projects/cm_mcu/LocalTasks.c

```diff
@@ -656,3 +656,136 @@ void init_registers_ff()
   apollo_i2c_ctl_reg_w(3, 0x21, 1, 0x03, 1, 0x01); //  00000001 [P17..P10]
 }
 #endif // REV2
+
+#ifdef REV2
+
+static int load_register(int reg_count, uint16_t reg_page, uint16_t i2c_addrs)
+{
+  uint32_t triplet; // two-byte address and data read from eeprom
+  uint32_t reg0; // a high byte of two-byte address (a page of clock config to keep track when writing data to a clock chip)
+  uint32_t reg1; // a low byte of two-byte address
+  uint32_t data; // data from a config file for each address
+  int HighByte = -1; // keep track when reg0 is changed
+  int status = -10; // write data to a clock chip failed or not
+  uint16_t reg_pages[3] = {0,0,0}; // an array of pages the triplet is on in eeprom for an edge case
```

Comment:
That sounds good! one page actually has 128 registers so it might be voiding the address 0x007F and 0x007E instead. I will have to modify the writing script and retesting the workflow again, but this way seems to be more sustainable, so I can do it.

Reply:
done